pretyping/cases.ml

```diff
@@ -46,8 +46,8 @@ module NamedDecl = Context.Named.Declaration
 type pattern_matching_error =
   | BadPattern of constructor * constr
   | BadConstructor of constructor * inductive
-  | WrongNumargConstructor of constructor * int
-  | WrongNumargInductive of inductive * int
+  | WrongNumargConstructor of constructor * bool * int * int * int
+  | WrongNumargInductive of inductive * bool * int * int * int
```

Comment:
Can we get inline records? This is pretty cryptic at first sight.

Reply:
Done. Are the field names informative enough to you?